packages/mobile/e2e/src/usecases/ResetAccount.js

```diff
@@ -1,26 +1,38 @@
-import { enterPinUiIfNecessary, waitForElementId, sleep } from '../utils/utils'
+import { enterPinUiIfNecessary, pixelDiff, waitForElementId } from '../utils/utils'
 import { SAMPLE_BACKUP_KEY } from '../utils/consts'
+import { dismissBanners } from '../utils/banners'
 
 export default ResetAccount = () => {
+  beforeEach(async () => {
+    await device.reloadReactNative()
```

Comment:
do we need this?

Reply:
We do need this with the new test structure so the tests start off in a known state. `await device.reloadReactNative()` will launch the app at the homescreen for a restored account / at the start of education for an unrestored account.